variables.tf

```diff
@@ -285,6 +287,18 @@ variable "operating_system" {
   }
 }
 
+variable "pod_subnet" {
```

Comment:
Can we give these a more descriptive name if they need to be cidr ranges? like "pod_subnet_cidr"?